lib/classes/config.js

```diff
@@ -12,7 +11,7 @@ class Config {
   }
 
   update(config) {
-    return _.merge(this, config);
+    return Object.assign(this, config);
```

Comment:
Did you validate that it's going to be a safe change? It seems like a pretty generic method where deep vs shallow copy behavior might have an impact